packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -283,6 +283,13 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
   _RenderEditableCustomPaint? _foregroundRenderObject;
   _RenderEditableCustomPaint? _backgroundRenderObject;
 
+  @override
+  void dispose() {
+    _foregroundRenderObject?.dispose();
+    _backgroundRenderObject?.dispose();
+    super.dispose();
```

Comment:
why not null out the references as well as disposing?

Reply:
Done